packages/spectral-config/package.json

```diff
@@ -2,12 +2,16 @@
   "name": "@readme/spectral-config",
   "version": "3.0.6",
   "description": "ReadMe coding standards for API documentation",
-  "main": "src/index.js",
+  "main": "dist/index.cjs",
```

Comment:
do you need the `exports: {}` stuff here?

Reply:
i don't think so, i've been QA'ing using the `readme-metrics-api` main repo it works with this spectral config:

<img width="319" alt="CleanShot 2023-09-08 at 14 28 34@2x" src="https://github.com/readmeio/standards/assets/8854718/c70c0fe0-8a04-4b7d-a88a-be8be750bcba">